conversation/v1.ts

```diff
@@ -64,18 +65,17 @@ class ConversationV1 extends GeneratedConversationV1 {
     const missingParams = getMissingParams(params, requiredParams);
     if (missingParams) { return callback(missingParams); }
 
-    const _params: GeneratedConversationV1.UpdateIntentParams = {
-      workspace_id: params.workspace_id,
-      intent: params.old_intent,
-      new_intent: params.intent,
-      new_description: params.description,
-      new_examples: params.examples
-    };
+    const _params = extend({}, params);
+    _params.intent = params.old_intent;
+    _params.new_intent = params.intent;
+    _params.new_description = params.description;
+    _params.new_examples = params.examples;
```

Comment:
The problem is that you will still send the old parameters along with the new parameters. You should not send the old parameters.